### PR TITLE
build: get ci green

### PR DESCRIPTION
**Pull request by davidjgoss**
TBA

### CODE REVIEW

**Comment by mattwynne** [comment]
I have no idea how the `*-testdata` folders are supposed to be populated; needs more digging into the makefiles. I don't think we can call this done until those are working.

**Comment by mattwynne** [comment]
re (2) - TIL https://unix.stackexchange.com/questions/517190/what-causes-make-to-delete-intermediate-files

**Comment by mattwynne** [comment]
I've reworded the ruby-testdata Makefile a good deal to simplify it:

* copies the `ndjson` files from out of the CCK
* run `cucumber-ruby` directly in the CCK folder, so the attachments can remain in there.
* keep a list of "unsupported" CCK features, rather than having stub make targets for them.

I think there's more we can do. I'd like to see all the `default.mk` files go away, and rename the `ruby-testdata` folder to `testdata`

There's also a bunch of clutter in that folder, like a `Rakefile` which runs `rspec` even though as far as I can tell there's no actual Ruby code in there?!

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+      - run: cd go && make
```
Comment (aurelien-reeves) [inline .github/workflows/test.yaml:19]:
```suggestion
      - name: make
        run: make
        working-directory: go
```

**Review by aurelien-reeves — COMMENTED** [review]
Looks good

**Comment by davidjgoss** [comment]
Nicely done @mattwynne!

**Pull request merged**